### PR TITLE
24.1.3 Enterprise patch release guidance and changelog

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Removes 24.1.1 upgrade warning from main Enterprise release notes. 
Adds separate Enterprise changelog
Adds Enterprise upgrade guide page, stripping out most upgrade guidance from the release notes page.
Adds 24.1.3 changelog entries.
Adds custom config and admonitions for 24.1.2 upgrade.

### CODE REVIEW

File: platform_versioned_docs/version-24.1/enterprise/release_notes/enterprise_changelog.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+- Fix: Disable launch form submit button during form validation
+- Fix: Allow workDir edit during pipeline launch 
+- Fix: Data Explorer preview for Azure Blob Storage files
+- Fix: GCP resource leak 
+- Fix: Remote config read issue
+- Fix: Dependency vulnerabilities 
+- Improvement: Add tag propagation for storage volumes to AWS launch template
+- Improvement: Implement custom H8 2nd-level cache to replace Redisson cache 
+- Improvement: Extend "determineTextFormat" helper function
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1/enterprise/release_notes/enterprise_changelog.mdx:18]:
```suggestion
- Improvement: Extend `determineTextFormat` helper function 
```

File: platform_versioned_docs/version-24.1/enterprise/release_notes/enterprise_latest.mdx [hunk 2]
```diff
@@ -79,68 +74,25 @@ The property that determines the expiration time of the JWT access token (used f
...
-If you have any further questions, please [contact support](https://support.seqera.io).
-:::
-
-This version requires a database schema update. Make a backup of your Platform database prior to upgrade. 
+## Upgrade steps 
 
-:::info
-If you are upgrading from a version older than 23.4.1, update your installation to version [23.4.4](./23.4.mdx) first, before updating to 24.1.1 with the steps below.
-:::
+1. This version includes an update to the Platform Enterprise H8 cache. **Do not start the upgrade while any pipelines are in a running state as active run data may be lost.**
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1/enterprise/release_notes/enterprise_latest.mdx:79]:
```suggestion
1. This version includes an update to the Platform Enterprise H8 cache. **Do not start the upgrade while any pipelines are in a `running` state as active run data may be lost.**
```

**Review by justinegeffen — APPROVED** [review]
Two minor comments but otherwise LGTM, thank you!

File: platform_versioned_docs/version-24.1/enterprise/release_notes/enterprise_changelog.mdx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+- Fix: Disable launch form submit button during form validation
+- Fix: Allow workDir edit during pipeline launch 
+- Fix: Data Explorer preview for Azure Blob Storage files
+- Fix: GCP resource leak 
+- Fix: Remote config read issue
+- Fix: Dependency vulnerabilities 
+- Improvement: Add tag propagation for storage volumes to AWS launch template
+- Improvement: Implement custom H8 2nd-level cache to replace Redisson cache 
+- Improvement: Extend `determineTextFormat` helper function
```
Comment (tcrespog) [inline platform_versioned_docs/version-24.1/enterprise/release_notes/enterprise_changelog.mdx:18]: New addition included in `24.1.3-RC1` that wasn't present in `24.1.2`: `nf-launcher` has been [bumped](https://github.com/seqeralabs/platform/commit/97eb7fe5e5c6aaf100d8a4ad43aef65225648885).

**Review by tcrespog — APPROVED** [review]
Added a comment: https://github.com/seqeralabs/docs/pull/182#discussion_r1728389027. Other than that, looks good to me.

File: platform_versioned_docs/version-24.1/enterprise/release_notes/enterprise_changelog.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
+---
+title: " Enterprise changelog"
+description: "Seqera Platform Enterprise changelog"
+date: "4 July 2024"
+tags: [changelog]
+---
+
+### 24.1.3 Enterprise - 22 August 2024
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/enterprise/release_notes/enterprise_changelog.mdx:8]:
```suggestion
### 24.1.3 Enterprise - 23 August 2024
```

File: platform_versioned_docs/version-24.1/enterprise/upgrade.mdx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+1. If you're using the pipeline optimization service (`groundswell` database) in a database separate from your Seqera database, update the MySQL image for your `groundswell` database instance while the application is down (during step 4 or 5 above). If you're using the same database instance for both, the `groundswell` update will happen automatically during the Seqera database update.
+
+### Custom deployments
+
+- Run the `/migrate-db.sh` script provided in the `migrate-db` container. This will migrate the database schema.
+- Deploy Seqera following your usual procedures.
+
+### Nextflow launcher image
+
+If you must host your nf-launcher container image on a private image registry, copy the [nf-launcher image](https://quay.io/seqeralabs/nf-launcher:j17-24.04.2) to your private registry. Then update your `tower.env` with the launch container environment variable:
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/enterprise/upgrade.mdx:36]:
```suggestion
If you must host your nf-launcher container image on a private image registry, copy the [nf-launcher image](https://quay.io/seqeralabs/nf-launcher:j17-24.04.4) to your private registry. Then update your `tower.env` with the launch container environment variable:
```

**Pull request merged**